packages/faceted-search/src/components/facetedSearch.propTypes.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+
+const badgeDefinitionRawDataPropTypes = PropTypes.shape({
+	attribute: PropTypes.string.isRequired,
+	label: PropTypes.string.isRequired,
+	operators: PropTypes.arrayOf(PropTypes.string),
+	type: PropTypes.string.isRequired,
+	metadata: PropTypes.shape({
+		badges_per_facet: PropTypes.string,
```

Comment:
What is this key?

Reply:
nb of badges allowed per facet. For example you can have multiple badge name, but only one connection type.